src/test/ui/rfc-2565-param-attrs/param-attrs-builtin-attrs.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(param_attrs)]
+
+FIXME
+pub fn _second(#[inline] _bar: u32) {}
```

Comment:
`unused` warnings are disabled in UI tests by default, so you can avoid the underscores in names.

Also, could you add `FIXME`s as comments, so they don't change test behavior?